EasyPost/VersionInfo.cs

```diff
@@ -0,0 +1,5 @@
+﻿using System.Reflection;
```

Comment:
Can we move all the helpful `VersionAssembly` comments from their individual locations into this file?

```
// Version information for an assembly consists of the following four values:
//
//      Major Version
//      Minor Version
//      Build Number
//      Revision
//
// You can specify all the values or you can default the Build and Revision Numbers
// by using the '*' as shown below:
// [assembly: AssemblyVersion("1.0.*")]
```